apps/runtime-demo/3005-runtime-host/src/App.tsx

```diff
@@ -50,14 +19,102 @@ function RemoteButton() {
   );
 }
 
+const WebpackSvgRemote = lazy(() =>
+  import('remote1/WebpackSvg').then((m) => {
+    return m;
+  }),
+);
+
+const WebpackPngRemote = lazy(() => import('remote1/WebpackPng'));
+
 const App = () => (
   <div>
-    <h1>Bi-Directional</h1>
-    <h2>App 1</h2>
-    <LocalButton />
-    <React.Suspense fallback="Loading Button">
-      <RemoteButton />
-    </React.Suspense>
+    <h1>Runtime Demo</h1>
+    <h2>Host</h2>
+    <h3>check static remote</h3>
+    <table border={1} cellPadding={5}>
+      <thead>
+        <tr>
+          <td></td>
+          <td>Test case</td>
+          <td>Expected</td>
+          <td>Actual</td>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>✅</td>
```

Comment:
Can we tell if the content is equal to show the correct and wrong status

Reply:
shall we hide the warning ?